euth_wagtail/assets/scss/core/_bootstrap-variables.scss

```diff
@@ -7,6 +7,7 @@ $gray-dark:             #2b2b2b;
 $gray:                  #818181;
 $gray-light:            #d8d8d8;
 $gray-lighter:          #f4f4f4;
+$gray-muted:            #6c757d;
```

Comment:
Why do we need a new colour? Can't we just use the `$gray`?

Reply:
actually I was unsure here and wanted to ask, but forgot. Thanks for the catch and change @fuzzylogic2000